src/System.Data.SqlClient/tests/ManualTests/DataCommon/DataTestUtility.cs

```diff
@@ -74,6 +74,20 @@ public static bool IsLocalDBInstalled()
             return false;
         }
 
+        public static bool IsIntegratedSecuritySetup()
+        {
+            string integratedSecuritySetupFlag = Environment.GetEnvironmentVariable("TEST_INTEGRATEDSECURITY_SETUP");
+            if (!string.IsNullOrWhiteSpace(integratedSecuritySetupFlag))
+            {
+                int result;
+                if (int.TryParse(integratedSecuritySetupFlag.Trim(), out result))
```

Comment:
Nit: `if (int.TryParse(integratedSecuritySetupFlag.Trim(), out int result))`

Reply:
And remove `int result;`

Reply:
Also Trim() is not needed on the integratedSecuritySetupFlag. int.TryParse trims the spaces.

Reply:
You could condense this function to 
```C#
public static bool IsIntegratedSecurity () => int.TryParse(Environment.GetEnvironmentVariable("TEST_INTEGRATEDSECURITY_SETUP"), out int result) ? result == 1 : false;
```

Reply:
Excellent suggestion, done. I have condensed a similar method `IsLocalDBInstalled` as well.